apps/web/src/lib/api/external/laskugeneraattori/actions.ts

```diff
@@ -0,0 +1,116 @@
+import {
+  type InvoiceGeneratorFormState,
+  type LaskugeneraattoriRequest,
+} from "./index";
+
+// https://stackoverflow.com/a/22015930
+const zip = <T>(...arr: T[][]) =>
+  Array(Math.max(...arr.map((a) => a.length)))
+    .fill(0)
+    .map((_, i) => arr.map((a) => a[i]));
+
+export const laskugeneraattoriBaseUrl =
```

Comment:
This needs to not be exported once the file is change to "use server", server actions files can't export strings, only async functions.